nxdrive/metrics.py

```diff
@@ -0,0 +1,48 @@
+from logging import getLogger
+from queue import Empty, Queue
+from typing import TYPE_CHECKING, Any, Dict
+
+from .constants import TIMEOUT
+from .engine.workers import Worker
+from .exceptions import ThreadInterrupt
+from .options import Options
+
+log = getLogger(__name__)
+
+_ME_ENDPOINT = "/site/api/v1/me"
```

Comment:
No need to be so specific:
```suggestion
_ENDPOINT = "/site/api/v1/me"
```